exercises/bowling/bowling_test.exs

```diff
@@ -0,0 +1,132 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("example.exs")
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule BowlingTest do
+  use ExUnit.Case
+
+  defp roll_from_list(game, []) do game end
+
+  defp roll_from_list(game, [roll | rolls]) do
+    roll_from_list(Bowling.roll(game, roll), rolls)
+  end
+
+  test "can score all 0s" do
+    game = Bowling.roll(0)
+    rolls = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
Wondering if it would be nice/helpful to format this to display the frames? 

(Assuming `roll/1` changes per my comment it could look like this?)
```
rolls = [
 0, 0,
 0, 0,
 0, 0,
 0, 0,
 0, 0,
 0, 0,
 0, 0,
 0, 0,
 0, 0,
 0, 0
]
```